aggregator/src/binaries/janus_cli.rs

```diff
@@ -54,6 +65,86 @@ pub async fn run(command_line_options: CommandLineOptions) -> Result<()> {
 
 #[derive(Debug, Parser)]
 enum Command {
+    /// Generates & writes a new global HPKE key.
+    GenerateGlobalHpkeKey {
+        #[clap(flatten)]
+        kubernetes_secret_options: KubernetesSecretOptions,
+
+        /// Numeric identifier of the HPKE configuration to generate
+        #[arg(long)]
+        id: u8,
+
+        /// HPKE Key Encapsulation Mechanism algorithm
+        #[arg(long)]
+        kem: KemAlgorithm,
+
+        /// HPKE Key Derivation Function algorithm
+        #[arg(long)]
+        kdf: KdfAlgorithm,
+
+        /// HPKE Authenticated Encryption with Associated Data algorithm
+        #[arg(long)]
+        aead: AeadAlgorithm,
+
+        /// The location to write the encoded HpkeConfig
+        #[arg(long)]
+        hpke_config_out_file: Option<PathBuf>,
+    },
+
+    /// Sets the state of a global HPKE key.
+    SetGlobalHpkeKeyState {
+        #[clap(flatten)]
+        kubernetes_secret_options: KubernetesSecretOptions,
+
+        /// Numeric identifier of the HPKE configuration to modify
+        #[arg(long)]
+        id: u8,
+
+        /// State to change the HPKE key to
+        #[arg(long)]
+        state: HpkeKeyState,
+    },
+
+    /// Adds a taskprov peer aggregator.
+    AddTaskprovPeerAggregator {
+        #[clap(flatten)]
+        kubernetes_secret_options: KubernetesSecretOptions,
+
+        /// The peer's endpoint, as a URL.
+        #[arg(long)]
+        peer_endpoint: Url,
+
+        /// This aggregator's role.
+        #[arg(long)]
+        role: Role,
+
+        /// The taskprov verify_key_init value, in unpadded base64url.
+        #[arg(long, env = "VERIFY_KEY_INIT")]
```

Comment:
```suggestion
        #[arg(long, env = "VERIFY_KEY_INIT", hide_env_values = true)]
```
(ouch, there's other secrets here that we don't hide_env_values for. I can run through these in another PR, or you can if you want)